modules/cluster-logging-elasticsearch-rules.adoc

```diff
@@ -13,38 +13,54 @@ You can view these alerting rules in Prometheus.
 |Description
 |Severity
 
+|ElasticsearchBulkRequestsRejectionJumps
+|Elasticsearch is experiencing an increase in bulk rejections on the specified node. This node may not be keeping up with the indexing speed."
+|Warning
+
 |ElasticsearchClusterNotHealthy
-|Cluster health status has been RED for at least 2m. Cluster does not accept writes, shards may be missing or master
+|The cluster health status has been RED for at least 2m. The cluster does not accept writes, shards may be missing, or themaster
  node hasn't been elected yet.
 |critical
 
 |ElasticsearchClusterNotHealthy
-|Cluster health status has been YELLOW for at least 20m. Some shard replicas are not allocated.
+|The cluster health status has been YELLOW for at least 20m. Some shard replicas are not allocated.
 |warning
  
-|ElasticsearchBulkRequestsRejectionJumps
-|High Bulk Rejection Ratio at node in cluster. This node may not be keeping up with the indexing speed.
-| warning
+|ElasticsearchDiskSpaceRunningLow
+|The cluster is expected to be out of disk space within the next 6 hours.
+|Critical
+
+|ElasticsearchHighFileDescriptorUsage
+|The cluster is predicted to be out of file descriptors within the next hour
+|warning
+
+|ElasticsearchJVMHeapUseHigh
+|The JVM Heap usage on the specified node is high.
+|Alert
 
 |ElasticsearchNodeDiskWatermarkReached
-|Disk Low Watermark Reached at node in cluster. Shards can not be allocated to this node anymore. You should
+|The specified node has hit the low watermark due to low free disk space. Shards can not be allocated to this node anymore. You should
```

Comment:
```suggestion
|The specified node has hit the low watermark due to low free disk space. Shards can not be allocated to this node anymore.
```
Also, consider using a different term than watermark in both this entry and the next.